tests/benchmark/stateful/bloatnet/depth_benchmarks/test_worst_case_attack_execute.py

```diff
@@ -0,0 +1,539 @@
+"""
+abstract: BloatNet worst-case attack benchmark for maximum SSTORE stress (Execute Mode Version).
+
+This test implements a worst-case scenario for Ethereum block processing that exploits
+the computational complexity of Patricia Merkle Trie operations. It uses CREATE2 to deploy
+contracts at pre-mined addresses with shared prefixes, maximizing trie traversal depth.
+
+Key features:
+- Deploys 15,000 contracts via CREATE2 to addresses with configurable prefix sharing depth
+- Each contract has deep storage slots with configurable trie depth
+- Executes optimized attack bytecode that performs multiple SSTORE operations
+- Respects EIP-170 (24KB contract size) and Fusaka (16M gas per tx) limits
+- Verifies that deep storage slots are correctly modified
+
+Test parameters:
+- storage_depth: Depth of storage slots in the contract (e.g., 10)
+- account_depth: Depth of account address prefix sharing (e.g., 5)
+- Fixed 15,000 contracts deployed/attacked
+- Gas per attack call: 50,000 (sufficient for cold SSTORE operations)
+- Compilation requires: solc --metadata-hash none (for reproducible bytecode)
+"""
+
+import json
+import pytest
+from pathlib import Path
+from subprocess import run, PIPE
+from execution_testing import (
+    Account,
+    Alloc,
+    Block,
+    BlockchainTestFiller,
+    Bytecode,
+    Fork,
+    Op,
+    Transaction,
+    Address,
+)
+
+# Attack function selector for WorstCaseERC20.attack(uint256)
+ATTACK_SELECTOR = 0x64dd891a  # attack(uint256) - verified with: cast sig "attack(uint256)"
+
+# Maximum gas per transaction (Fusaka EIP limit)
+MAX_GAS_PER_TX = 16_000_000
+
+NUM_CONTRACTS = 15000  # Full test with 15,000 contracts
+
+# Nick's deterministic deployer address (must be pre-deployed in execute mode)
+NICK_DEPLOYER = Address("0x4e59b44847b379578588920ca78fbf26c0b4956c")
```

Comment:
FYI, this is not Nick's deployer, it is the deterministic deployment: https://github.com/Arachnid/deterministic-deployment-proxy/blob/master/source/deterministic-deployment-proxy.yul

That one was deployed using Nick's method - it is creating a transaction without a private key. This ensures that the transaction which we "send" cannot be overwritten (because no one has the key) and we can thus deterministically deploy a contract on most non-mainnet chains (and of course mainnet). Nick's method: https://medium.com/patronum-labs/nicks-method-ethereum-keyless-execution-168a6659479c